test/falsy-truthy.js

```diff
@@ -0,0 +1,33 @@
+const assert = require("assert");
+
+describe("Falsy values", function () {
+  it("Should return a complete list of falsy values.", function () {
+    assert.strictEqual(isValueFalsy(false), true);
+    assert.strictEqual(isValueFalsy(0), true);
+    assert.strictEqual(isValueFalsy(""), true);
+    assert.strictEqual(isValueFalsy(null), true);
+    assert.strictEqual(isValueFalsy(undefined), true);
+    assert.strictEqual(isValueFalsy(NaN), true);
+  });
+  it("Should return a list of truthy values.", function () {
+    assert.strictEqual(isValueFalsy(true), false);
+    assert.strictEqual(isValueFalsy({}), false);
+    assert.strictEqual(isValueFalsy([]), false);
```

Comment:
This is the next most common source of bugs. Because an empty string is falsy, sometimes people incorrectly think an empty array or an empty object will be falsy as well.

Reply:
yea good to know about it. @josh-egan